etc/preprocess.py

```diff
@@ -35,7 +35,7 @@
     parser.add_argument('--summarize', dest='summarize', action='store_true', default=False,
                         help="summarize code blocks with SLOC count")
     parser.add_argument('filename', metavar='FILE', action='store')
-    args = parser.parse_args()
+    argv = parser.parse_args()
```

Comment:
I think this one should remain `args`: same logic as for codebasin.py and coverage.py.